Library/Power Ups/Power Ups 8 Limitations Limitation Modifiers.adm

```diff
@@ -40,6 +40,158 @@
 			"reference": "PU8:11",
 			"cost": -25
 		},
+		{
+			"id": "a6c73279-9faf-432b-949a-e8484a65af09",
+			"type": "modifier",
+			"name": "Requires Gestures",
+			"reference": "PU8:5",
+			"cost": -10
+		},
+		{
+			"id": "bff6123d-7988-4da8-86b5-02985db19e33",
+			"type": "modifier",
+			"name": "Requires Magic Words",
+			"reference": "PU8:5",
+			"cost": -10
+		},
+		{
+			"id": "146fdf4b-c18f-4b38-a86e-c0efe0d2205f",
+			"type": "modifier",
+			"name": "Requires Material Component",
+			"reference": "PU8:5",
+			"cost": -10
+		},
+		{
+			"id": "3e45724b-b51f-4f57-9d17-41ffadf383f3",
+			"type": "modifier",
+			"name": "Requires Material Component",
+			"reference": "PU8:5",
+			"notes": "Extermely Rare",
+			"cost": -15
+		},
+		{
+			"id": "19558575-7d5c-47de-9f97-33278c806739",
+			"type": "modifier",
+			"name": "Requires Simple Ritual",
+			"reference": "PU8:5",
+			"notes": "1d minutes of quiet chanting and simple materials",
+			"cost": -5
+		},
+		{
+			"id": "efaeec94-175b-4051-b659-5d0343bd5f0c",
+			"type": "modifier",
+			"name": "Requires Typical Ritual",
+			"reference": "PU8:5",
+			"notes": "1dx5\rminutes and @cumbersome preparation materials or successful casting of relevant spell (specify spell)@",
```

Comment:
This should not have a '\r' in it. GCS uses only line feed '\n' for embedded returns. If this was generated by GCS, it's a bug. Are you running on Windows?

Reply:
I am running windows, yes.
This was generated by GCS as well, yes. Very odd. This is what it appears like in GCS.
![image](https://github.com/richardwilkes/gcs_master_library/assets/1914417/e7065048-8bc0-487a-b81d-50f6261ed9ca)

Reply:
Interesting. Did you copy & paste that from elsewhere, perhaps? I bet I don't filter out invisible characters and I probably should.